src/main/java/Ebot.java

```diff
@@ -0,0 +1,256 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+
+public class Ebot {
+    public static final String OUTPUT_FILE_NAME = "./data/ebot.txt";
+    private static final String LINE_DIVIDER = ("___________________________________________");
+    private static final String BIG_INDENT = "\t\t";
+    private static final String SMALL_INDENT = "\t";
+
+    public static void main(String[] args) {
+        String welcomeMessage = (SMALL_INDENT + "Hello! I'm Ebot" +
+                System.lineSeparator() +
+                SMALL_INDENT + "What can i do for you?");
+        String exitMessage = (SMALL_INDENT + "Bye. Hope to see you again soon!");
+
+        // Program start, initialize scanner for user inputs
+        String userInput;
+        Scanner in = new Scanner(System.in);
+
+        // Initialize list
+        int numberOfLists = TaskList.getNumberOfTaskLists();
+        String taskListName = ("list") + numberOfLists;
+        TaskList taskList = new TaskList(taskListName);
+
+        // Initialize file
+        createFile();
+
+        // Print Welcome Message
+        System.out.println(LINE_DIVIDER + System.lineSeparator() + welcomeMessage);
+        System.out.println(LINE_DIVIDER);
+
+        // User chat loop
```

Comment:
You don't need to write comments for certain snippets if the code is already clear